labelbox/schema/benchmark.py

```diff
@@ -3,12 +3,21 @@
 
 
 class Benchmark(DbObject):
-    """ Benchmarks (also known as Golden Standard) is a quality assurance tool
-    for training data. Training data quality is the measure of accuracy and
-    consistency of the training data. Benchmarks works by interspersing data
+    """ Represents a benchmark label.
+    
+    The Benchmarks tool works by interspersing data
     to be labeled, for which there is a benchmark label, to each person labeling.
     These labeled data are compared against their respective benchmark and an
     accuracy score between 0 and 100 percent is calculated.
+
+    Attributes:
+        created_at (DateTime)
```

Comment:
Another issue with class names. Like with `str`, these attributes should probably be declared as python classes they belong to, namely `datetime`, `int` and `float`.

I know it's confusing as I'm declaring class variables like for example `Field.DateTime`, but that's a different thing.

Reply:
Reverted back to str, datetime, int, and float.